privacy-policy/index.md

```diff
@@ -0,0 +1,117 @@
+
+This document details the data the Vox Pupuli collects about you, why it is
+collected, what is done with it, and your options for how it's processed.
+
+The Vox Pupuli project does not share data with 3rd parties, except for hosting
+requirements (e.g. GitHub, YouTube) as detailed below, or when required to by
+law.
+
+# Types of data we collect
+
+* Contact details
+  * Name
+  * E-Mail address
+
+# How and why we use your data
+
+## The website
+
+The Vox Pupuli website is a static site that does not track, collect, or store 
+infromation or set cookies. There is no account needed or required to access 
+the website itself. The website itself is hosted with [Github Pages](https://pages.github.com/) 
+and any data gathered by Github will adhere to the [Github Privacy Policy](https://help.github.com/articles/github-privacy-statement/).
+
+The website does host a community run Blog series and do require that the 
+author's name be visible in the blog post indefinitely.
+
+## The codebases
+
+The Vox Pupuli community uses [GitHub](https://github.com) to manage the code 
+that comprises the various Puppet modules, plugins, tools, and RubyGems the 
+community is responsible for maintaining. Refer to [GitHub's Privacy Policy](https://help.github.com/articles/github-privacy-statement/)
+for information on how GitHub handles the data you post there. We also use 
+non-identifiable data from the codebases to provide aggregated statistics such 
+as total member count, number of open PRs, total number of projects, etc.
+
+Please note that due to the open source nature of the community, we may retain 
+limited personally-identifiable data indefinitely. 
+
+* Legal basis for this data usage: Legitimate interest
+
+## Communication
+
+The Vox Pupuli project maintains two public IRC channels on 
+[Freenode](https://freenode.net), refer to [Freenode's privacy policy](https://freenode.net/policies#privacy-policy-and-data-retention)
+for details on how they handle your data. Our bot keeps a rolling backlog in 
+the IRC channel for logging purposes, however that backlog is rolled off after
+10 minutes.
+
+The Vox Pupuli community also uses a mailing list via the 
+[Groups.io](https://groups.io) website. When signing up for a Groups.io 
+account, you provide an E-Mail address and password. The email address is used
+to log into the web interface for the mailing list and to send updates to your 
+mailing list topics, personal messages from other users, and summaries. You have 
+full control over the emails sent from your Groups.io account preferences. 
+Please see the [Groups.io Privacy Policy](https://groups.io/static/privacy) on 
+any additional data usage relating to your account.
+
+* Legal basis for this data usage: Legitimate interest, Consent
+
+# Your rights regarding your data
+
+## Your choices
+
+### You can choose not to provide us with personal data
+
+Using a VPN, Tor, or other IP/UserAgent masking servers will not affect your
```

Comment:
Not negatively? Obviously there's the effect of hiding your IP/user agent. Using Tor can negatively affect it as CloudFlare has been known to make life harder for Tor users (https://support.cloudflare.com/hc/en-us/articles/203306930-Does-CloudFlare-block-Tor-)